src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -24,11 +25,13 @@ public void addSession(MovieSession movieSession, User user) {
         newTicket.setUser(user);
         newTicket.setCinemaHall(movieSession.getCinemaHall());
         newTicket.setShowTime(movieSession.getShowTime());
-        newTicket.setMovie(movieSession.getMovie());
+        newTicket.setMovieSession(movieSession);
+        ticketDao.add(newTicket);
 
-        ShoppingCart shoppingCart = shoppingCartDao.getByUser(user);
-        shoppingCart.getTickets().add(ticketDao.add(newTicket));
-        shoppingCartDao.update(shoppingCart);
+        ShoppingCart shoppingCart = new ShoppingCart();
+        shoppingCart.setUser(user);
+        shoppingCart.setTickets(List.of(newTicket));
+        shoppingCartDao.add(shoppingCart);
```

Comment:
What happened if we will create new shopping cart every time, when user put new ticket to cart?